packages/components/src/FormField/FormField.css

```diff
@@ -72,7 +72,6 @@
 
 textarea.formField {
   height: auto;
-  min-height: calc(var(--field--height) * 2);
```

Comment:
I'm guessing this still needs to be removed? Sorry I wasn't around when this change was made.

Reply:
Im pretty sure it does, in the case you want, `min-height` should be covered by the `rows` attribute